modules/cluster-entitlements.adoc

```diff
@@ -39,17 +39,19 @@ ifndef::openshift-origin[]
 
 In {product-title} {product-version}, you require access to the internet to
 ifndef::restricted[]
-install and
+install
 endif::restricted[]
 ifdef::restricted[]
-obtain the images that you require to install the cluster and to
+obtain the images that you require to install
 endif::restricted[]
-entitle your cluster.
-The Telemetry service, which runs by default to provide metrics about cluster health and the success of updates, also requires internet access. If your cluster is connected to the internet, Telemetry runs automatically, and your cluster is registered to the link:https://cloud.redhat.com/openshift[{cloud-redhat-com}]. From there, you can allocate entitlements to your cluster.
+your cluster.
+The Telemetry service, which runs by default to provide metrics about cluster health and the success of updates, also requires internet access. If your cluster is connected to the internet, Telemetry runs automatically, and your cluster is registered to the link:https://cloud.redhat.com/openshift[{cloud-redhat-com}].
+
+Once you confirm that your {cloud-redhat-com} inventory is correct, either from Telemetry or that which you manually created, link:https://access.redhat.com/documentation/en-us/subscription_central/2020-04/html/getting_started_with_subscription_watch/con-how-to-select-datacollection-tool_assembly-requirements-and-your-responsibilities-ctxt#red_hat_openshift[use subscription watch] to track your {product-title} subscriptions at the account or multi-cluster level.
```

Comment:
@lbarbeevargas, I wonder what _or that which you manually created_ means? Maybe if I knew more about Telemetry, it would be obvious.

Reply:
@jboxman I can try and get clarification on the "manually created" part. I pulled the info from a comment by a subscription watch PM (https://bugzilla.redhat.com/show_bug.cgi?id=1829099#c1).